doc/doxygen/install/install-linux.doxygen

```diff
@@ -2,7 +2,8 @@
 
   @page install_linux Building OpenMS on GNU/Linux
 
-  This document describes how to build OpenMS on a GNU/Linux system from source.
+  This document describes how to build OpenMS on a GNU/Linux system from
+  source. It assumes that you have root privileges on your system. 
```

Comment:
but you describe how to install without root below...

So why not mention it here?